docs/src/structure-factor.md

```diff
@@ -122,7 +115,7 @@ configurations. Information about calculating instantaneous data from a dynamic
 structure factor can be found in the following section.
 
 The basic usage for the instantaneous case is very similar to the dynamic case,
-except one calls `InstantStructureFactor` instead of `DynamicStructureFactor`.
+except one calls `InstantStructureFactor` instead of `SampledCorrelations`.
```

Comment:
Unsure what to do about `InstantStructureFactor`

Reply:
Seems weird to me that calling `InstantStructureFactor` builds you a `SampledCorrelations` (none of those words are the same anymore)

Reply:
Agreed. I thought this was just where we left it in the thread yesterday.

I guess would could have kept the whole `Instant` vs `Dynamic` dichotomy: `SampledDynamicCorrelations` and `SampledInstantCorrelations`.

Reply:
How about `SampledCorrelations` and `InstantCorrelations`, where "sampling" refers to the dynamical sampling process?

Reply:
What are the primary use cases of `InstantCorrelations` right now? Also note the overlap with `print_wrapped_intensities`.

Reply:
Ok